extensions/applicationinsights-clickanalytics-js/README.md

```diff
@@ -1,8 +1,17 @@
 # Microsoft Application Insights JavaScript SDK - Click Analytics Plugin
 
-Click Analytics Plugin for the Application Insights Javascript SDK, enables the following:
+Click Analytics Plugin for the Application Insights Javascript SDK, enables automatic tracking of the click events on web pages based on `data-\*` meta tags.
+This plugin uses the `data-\*` global attributes to capture the click events and populate telemetry data.
 
-- Automatic tracking of the click events on web pages based on data meta tags.
+## Some insights to effectively use the plugin
+
+1.  If the clicked HTML element donot have the `data-\*` attribute and if the `useDefaultContentName` flag is set to true, the plugin captures the `id` and the standard HTML attribute for contentName.
+2.  `customDataPrefix` provided by user should always start with `data-`, for example `data-sample-`. The reasoning behind this is that, in HTML the `data-\*` global attributes form a class of attributes called custom data attributes, that allow proprietary information to be exchanged between the HTML and its DOM representation by scripts.
```

Comment:
It's also a case that older browsers (IE, Safari) will drop attributes that it doesn't understand on the floor, unless they start with data-